numpy/ma/core.pyi

```diff
@@ -429,6 +428,7 @@ def resize(x, new_shape): ...
 def ndim(obj): ...
 def shape(obj): ...
 def size(obj, axis=...): ...
+def diff(a, /, n=..., axis=..., prepend=..., append=...): ...
```

Comment:
Yeah, lets go with this.  I guess the original diff is more precise, but no point if this is the way everything is here.  Thanks.